### ISSUE TITLE
Add ability to view changes with pull-request

### ISSUE THREAD

**Issue by fingermark** [github/hub#1409]
I just discovered `hub` and like the idea of submitting pull requests via cli.  One thing I like about the web is that I can review the pull request changes at the same time I am submitting the request, which lets me make one last final review.

Is it possible to review the changes of a PR with `hub` before submitting the request, preferably with the editor used to generate the PR message?

**Comment by mislav** [comment]
No, this isn't possible right now with hub, but I presume you want a feature similar to `git commit -v`? I use that one all the time.
